cni/network/network.go

```diff
@@ -1404,3 +1427,14 @@ func convertCniResultToInterfaceInfo(result *cniTypesCurr.Result) network.Interf
 
 	return interfaceInfo
 }
+
+func findDefaultInterface(ipamAddResult IPAMAddResult) int {
+	for i, ifInfo := range ipamAddResult.interfaceInfo {
+		if ifInfo.NICType == cns.InfraNIC {
+			return i
+		}
+	}
+
+	// error or nil here
+	return -1
```

Comment:
We will need an err i suppose, as we are using the `int` for indexing afterwards. It will be out of bounds for `-1`

Reply:
I am leaving in `-1` as I don't want to pass in an erroneous, but useable index.